runtime/generate.go

```diff
@@ -30,7 +32,7 @@ func GenerateServerCfg(cfg *types.Runtime) (err error) {
 
 	// make some minor changes to the cfg before using it
 	adjustForOS(cfg.WorkingDir, cfg.Platform, cfg)
-	cfg.Echo = &echoMessage
+	file.WriteString(echoMessage)
```

Comment:
Why write directly to the file? Isn't `cfg.Echo` written later anyway?

Reply:
Just to note I added the `ignore` flag to the echo field to prevent it being written normally, and manually wrote it to the file. That is why it's like the way it is.

Reply:
It's still a hack, but you justified it.